server/persistence/mongodb/mongodb_service_interaction.go

```diff
@@ -85,19 +96,54 @@ func (s *interactionService) Create(pm interaction.PlayerMeans) error {
 		return err
 	}
 
-	_, err = s.col.InsertOne(s.ctx, pm)
-	return err
+	return mongo.WithSession(s.ctx, s.session, func(sc mongo.SessionContext) error {
+		ior, err := s.col.InsertOne(sc, pm)
+		if err != nil {
+			return err
+		}
+		if ior.InsertedID == nil {
+			// TODO not sure if this is the right thing to check
```

Comment:
we should update this comment: I don't know if it's the right thing still, but I don't want TODOs in the mongo code